src/raudio.c

```diff
@@ -1297,6 +1297,102 @@ Music LoadMusicStream(const char *fileName)
     return music;
 }
 
+// extension including period ".mod"
+Music LoadMusicStreamFromMemory(const char *fileType, unsigned char* data, int dataSize){
```

Comment:
Please, could you review curly brackets alignment to follow [raylib coding conventions](https://github.com/raysan5/raylib/wiki/raylib-coding-conventions)?